### ISSUE TITLE
rnpm-plugin-windows: initialization script

### ISSUE THREAD

**Issue by kevinvangelder** [Microsoft/react-native-windows#937]
There has been some limited discussion on the topic, but we want to have a clear path before Gant starts the actual code.

What I envision would be a shared initialization script (living under the existing rnpm-plugin-windows) that presents the users with the following options:
```
Which Windows platforms would you like to support?
1. Just UWP (Windows 10+)
2. Just WPF (Windows 7/8)
3. Both UWP and WPF
```

**Comment by rozele** [comment]
I don't like the idea of adding an prompt, it's really nice to have a single command and no interaction right now.  Other options to consider:
1. Add a new command (`react-native wpf`)
2. Add a flag to the existing command (`react-native windows --wpf`) <-- needs more thought

It's fine for the command to live in rnpm-plugin-windows, since that's just a command proxy package anyway.

**Comment by rozele** [comment]
@GantMan and I discussed options for this.  I think the easiest way to start is to create an entirely separate command:
* `react-native wpf` to create a new `wpf` folder, similar to the `windows` folder for UWP
* Publish `react-native-wpf`, personally I prefer an ad hoc approach of publishing `react-native-wpf` via a script that copies the necessary files into a staging packages directory.  This will prevent breaking changes that would be introduced by a `react-native-windows-core` shared package.

**Comment by GantMan** [comment]
### Update

I a functioning WPF generator that fits the plans stated above.  

* I've created a node module named `rnpm-plugin-wpf` which I've used to test everything.  This is a temporary step to assist.  `react-native wpf` is also added to the `rnpm-plugin-windows` node module, but that will only be live once we are 👍  with the results.
* With the provided `react-native wpf` command,  a WPF solution and folder is generated.  This appears to be working just great.

The one thing I need help with is `react-native run-wpf`.  I'll contact you to see about advancing those questions.

Once all the above is in place there is a good standing solution for people to create WPF projects.  If you'd like, we can expose that in a release.  I don't want the branch to stray too far or risk getting out stale often.

When we have verified there is a successful generator path for WPF (merged or not), I can then begin the `react-native-windows-core` modifications in a new branch, which gives me a clear way to test if WPF and UWP are continually functioning as expected.

**Comment by GantMan** [comment]
Ok, thanks for the meeting, now run-wpf is working.

I'll move the code to a PR where we can review.

**Comment by GantMan** [comment]
PR #954

**Comment by rozele** [comment]
@GantMan @kevinvangelder Can this issue be closed?

**Comment by kevinvangelder** [comment]
I think so.
